dbcfeederlib/canreader.py

```diff
@@ -83,44 +89,58 @@ def stop(self):
 
     def _process_can_message(self, frame_id: int, data: Any):
-        try:
-            message_def = self._mapper.get_message_for_canid(frame_id)
-            if message_def is not None:
-                decode = message_def.decode(bytes(data), allow_truncated=True)
-                if log.isEnabledFor(logging.DEBUG):
-                    log.debug("Decoded message: %s", str(decode))
-                rx_time = time.time()
-                for signal_name, raw_value in decode.items():  # type: ignore
-                    signal: cantools.database.Signal = message_def.get_signal_by_name(signal_name)
-                    if isinstance(raw_value, (int, float)):
-                        # filter out signals with values out of defined range
-                        # which is usually because the signal's value is unavailable
-                        # (represented as e.g. "all bits set")
-                        if signal.minimum is not None and raw_value < signal.minimum:
-                            log.debug(
-                                "discarding out-of-range value [signal: %s, min: %s, value: %s]",
-                                signal_name, signal.minimum, raw_value
-                            )
-                            continue
-                        if signal.maximum is not None and raw_value > signal.maximum:
-                            log.debug(
-                                "discarding out-of-range value [signal: %s, max: %s, value: %s]",
-                                signal_name, signal.maximum, raw_value
-                            )
-                            continue
-                    for signal_mapping in self._mapper.get_dbc2vss_mappings(signal_name):
-
-                        if signal_mapping.time_condition_fulfilled(rx_time):
-                            log.debug(
-                                "Queueing %s, triggered by %s, raw value %s",
-                                signal_mapping.vss_name, signal_name, raw_value
-                            )
-                            self._queue.put(VSSObservation(
-                                signal_name, signal_mapping.vss_name, raw_value, rx_time))
-                        else:
-                            log.debug(
-                                "Ignoring %s, triggered by %s, raw value %s",
-                                signal_mapping.vss_name, signal_name, raw_value
-                            )
-        except Exception:
-            log.warning("Error processing CAN message with frame ID: %#x", frame_id, exc_info=True)
+        message_def = self._mapper.get_message_for_canid(frame_id)
+        if message_def is not None:
+            try:
+                decode = message_def.decode(bytes(data), allow_truncated=True, decode_containers=True)
+            except Exception as e:
+                log.warning("Error processing CAN message with frame ID: %#x", frame_id, exc_info=True)
+                log.warning("Error:  ", e)
```

Comment:
This line gives a linter error (i think), duplicate as we already have the line above?

Reply:
how can this be reproduced locally? on my machine, `flake8` passes fine:
```
$ flake8 .
$
```

Reply:
This is not reported by flake8, it is `pylint` that we run from https://github.com/eclipse-kuksa/kuksa-can-provider/blob/main/.github/workflows/can-provider.yml#L122

Installed in CI by

`pip3 install --no-cache-dir -r requirements.txt -r requirements-dev.txt`

This is how I run it locally:

```
erik@debian4:~/kuksa-can-provider$ pylint -E dbcfeeder.py dbcfeederlib test
************* Module dbcfeederlib.canreader
dbcfeederlib/canreader.py:98:16: E1205: Too many arguments for logging format string (logging-too-many-args)
```